src/header/header.scss

```diff
@@ -264,5 +292,54 @@
         }
       }
     }
+
+    .iui-split-menu {
+      display: inline-flex;
+      justify-content: center;
+      @include iui-focus;
+
+      svg {
+        width: 16px;
+        @include themed {
+          fill: var(--iui-icons-color-actionable);
+        }
+      }
+
+      &.iui-active {
+        svg {
+          fill: var(--iui-icons-color-primary);
+        }
+      }
+
+      svg {
+        border-radius: 50%;
+        padding: $iui-xxs;
+      }
+
+      &:hover,
+      &:focus-visible {
+        svg {
+          @include themed {
+            background-color: rgba(
+              var(--iui-color-foreground-body-rgb),
+              var(--iui-opacity-5)
+            );
+          }
```

Comment:
We don't usually use css variables (i.e. `var` in our components) because they are not supported in IE. You are already inside the themed mixin, so just use the `t` syntax and remove `--`. See example:

https://github.com/iTwin/iTwinUI/blob/4b69a213920d4af9f1acee63d2f3f79840c06e68/src/style/theme.scss#L425-L429